build.sh

```diff
@@ -115,6 +115,8 @@ if hasArg -v; then
 fi
 if hasArg -g; then
     BUILD_TYPE=Debug
+    export SKBUILD_INSTALL_STRIP=false
+    export SKBUILD_CMAKE_BUILD_TYPE=""Debug""
```

Comment:
Why double double quotes?
```suggestion
    export SKBUILD_CMAKE_BUILD_TYPE="Debug"
```

Reply:
I just copied as-is from the scikit-build documentation

Reply:
It looks like a typo, at least other code, including in their own testing, just use `SKBUILD_CMAKE_BUILD_TYPE=Debug`.
Fixed